### DISCUSSION TITLE
Урон Ворона зашкаливает

### DISCUSSION THREAD

**Discussion by LDS1** [dosinabox/g2nr_unofficial_update#418] [General]
Ворон всегда такой сильный был? Ваншотит гг с 500+ здоровья и в легкой броне ОнД.

**Comment by LDS1** [comment]
> Нет. Ворон очень слабый. У него только молния чуть чуть наносит урона. Есть сейв? 

[raven_save.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/7698048/raven_save.zip)

**Reply by Kor-Angar** [reply]
Нормальные хар-ки у Ворона. Обычный удар не пробивает броню и наносит минимальный ущерб 5 единиц.
![20211212061704_1](https://user-images.githubusercontent.com/49514764/145698859-a6ad3b50-0f03-4891-b172-b0ac3904878c.jpg)

атака по Ворону приводит к срабатыванию молнии
![20211212061951_1](https://user-images.githubusercontent.com/49514764/145698907-e89eb5d9-871a-4bf2-8146-7e4b543b79a5.jpg)

**Reply by Kor-Angar** [reply]
Ваншотить это убийство с одного удара?

https://www.youtube.com/watch?v=QNJB4VUCK8Y

**Reply by Kor-Angar** [reply]
так мало силы иметь не прилично. хоть бы инносу помолился на силу в Монастыре.

**Reply by LDS1** [reply]
> так мало силы иметь не прилично. хоть бы инносу помолился на силу в Монастыре.

нормально, если скейлится от ловкости

**Reply by Kor-Angar** [reply]
а что такое "скейлится от ловкости" ?

**Reply by LDS1** [reply]
масштабируется урон от ловкости у ловкого оружия в моей сборке

**Reply by Kor-Angar** [reply]
о..

**Comment by LDS1** [comment]
Может позже всю сборку кину. Ворон выносит в одно касание.

**Reply by Kor-Angar** [reply]
так если у тебя сборка, то наверное надо писать автору сборки ;-)

**Reply by Kor-Angar** [reply]
ты в своей сборке можешь просто через марвин "вселится" в Ворона и посмотреть его статы и статы когтя белиара.

**Reply by LDS1** [reply]
"сборка" это стимфикс и несколько юнион плагинов популярных + графика. Я смотрел, Ворон вроде в порядке, характеристики меча как до молитвы заблокированы

**Reply by Kor-Angar** [reply]
отключай по одному юнион плагины и узнаешь в чем была проблема. после этого можно будет написать автору юнион плагина.
